processing-service-core/api/swagger/swagger.yaml

```diff
@@ -1255,9 +1255,10 @@ definitions:
       scope:  
         type: string  
         description: Defines the scope of use for the global config key. 
```

Comment:
Add something to this description to communicate what the specific scope value means e.g. "If the scope is set to 'TENANT' then it may be overridden by tenant configurations".